ui/src/components/Code/index.jsx

```diff
@@ -35,7 +35,10 @@ export default class Code extends Component {
 
   render() {
     const { children, language, className, ...props } = this.props;
-    const code = highlight(language, children, true).value;
+    const code = highlight(children, {
+      language,
+      ignoreIllegals: true,
+    });
```

Comment:
When I expand `Payload` on a [task](http://localhost:5080/tasks/a6Baio9sRHyEtLArH07GfQ), it displays `[object Object]` instead of the task definition with syntax highlighting. This should fix it:

```suggestion
    }).value;
```

Reply:
Thank you for correcting it.